less/home-page.less

```diff
@@ -1,29 +1,48 @@
 // Home Content
 
 .home {
-
   .intro {
+    .main-header-img {
+      width: 100%;
+      border-radius: 5px;
+    }
 
     h2 {
       padding: 15px;
     }
 
     p {
-      padding:0 15px;
+      padding: 0 15px;
     }
-  
   }
 
   .space-images {
     display: flex;
     flex-wrap: wrap;
     width: 100%;
+    @media @phone {
+      justify-content: center;
+    }
+
+    .sky {
+      @media @phone {
+        display: none;
+      }
+    }
+    .launch {
+      @media @phone {
+        display: none;
+      }
```

Comment:
Here, for `.sky` and `.launch`, you could combine their media queries:
```less
.sky, .launch {
      @media @phone {
        display: none;
      }
    }
```
in order to not repeat yourself!